### ISSUE TITLE
Installing proteuslib-dev via Ubuntu on Windows

### ISSUE THREAD

**Issue by adam-a-a** [nawi-hub/proteuslib#189]
I was trying to setup a Linux environment so that I could help troubleshoot OS-related solver infeasibilities. 

I followed the readthedocs instructions, taking this as a chance to test out the install procedure. Everything seemed to go well, but then I stumbled across a couple of errors after installing proteuslib (for developers). 

**First error:**
`pyomo.solvers:IPOPT.py:63 Could not locate the 'ipopt' executable, which is required for solver ipopt `

I think I resolved this by running `idaes get-extensions`

**Second error:**
```
ERROR    pyomo.opt:solvers.py:587 Solver (ipopt) returned non-zero return code (127)
ERROR    pyomo.opt:solvers.py:594 Solver log:
/root/.idaes/bin/ipopt: error while loading shared libraries: libgfortran.so.4: cannot open shared object file: No such file or directory
```
To get past this, I ran the following installations:
```
sudo apt-get install gcc-7 g++-7
sudo apt-get install gfortran-7
```
**Error I am troubleshooting now:**
```
ERROR    pyomo.opt:solvers.py:594 Solver log:
/root/.idaes/bin/ipopt: error while loading shared libraries: liblapack.so.3: cannot open shared object file: No such file or directory
```
I was thinking that these errors might just be a result of my own Linux setup, but I also thought that I should mention my issue here in case someone else tries this and faces similar struggles.

@lbianchi-lbl @ksbeattie @dangunter - any suggestions would be appreciated! In the meantime, I'll try to get this resolved.

**Comment by adam-a-a** [comment]
The solution: 
I had to run 
`sudo apt-get install libatlas-base-dev
`
in my root directory (running it previously in /mnt/c/my_proteus_directory did not work). Now all tests are passing when I run pytest.

**Comment by lbianchi-lbl** [comment]
Hi @adam-a-a, thanks for reporting this.

- I definitely agree that this was a good test for the documentation covering the case where users install ProteusLib by itself on a system where they haven't used IDAES before
  - The documentation should mention to run `idaes get-extensions` after installing `proteuslib`: I believe that for the vast majority of cases, this only needs to happen once on a given system, but it does result in not being able to use the tool and therefore should be mentioned in the docs
- For the other errors, I think (and, feel free to correct me here) this might be within the responsibilities of IDAES/idaes-ext
  - The questions we might ask ourselves at this point would be:
    - What's the best way of improving the user/developer experience in case these errors occur, and 
    - At which level (IDAES extensions, `idaes get-extensions`, a wrapper within `proteuslib`) these preliminary checks would be best implemented

**Comment by lbianchi-lbl** [comment]
Addressed by #202: the extra steps required on a fresh installation are mentioned in the installation docs.
